region/brinstar/green/Spore Spawn Room.json

```diff
@@ -99,6 +99,41 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 2],
+      "name": "Plasma SBA Kill from Above (Patient)",
+      "requires": [
+        {"notable": "Plasma SBA Kill from Above"},
+        "Plasma",
+        "canSpecialBeamAttack",
+        {"ammo": {"type": "PowerBomb", "count": 1}},
```

Comment:
```suggestion
        {"ammo": {"type": "PowerBomb", "count": 2}},
```
The `PowerBomb` in `canSpecialBeamAttack` doesn't use an actual PB. It's funky and could be changed there instead.